yaas/blueprint.py

```diff
@@ -1,4 +1,138 @@
 # coding: utf-8
 
+from __future__ import absolute_import
+from __future__ import print_function
+
+import argparse
+import inspect
+import json
+import pprint
+import requests
+import sys
+
+from . import config
+
+
 def command(parser, args):
-    pass
+    """ Interact with Ambari blueprints. """
+    subparsers = parser.add_subparsers(dest='subcommand')
+    subcommands = {
+        'add': _add,
+        'list': _list,
+        'remove': _remove,
+        'show': _show,
+        }
+    for name, fn in subcommands.items():
+        subparsers.add_parser(name, help=inspect.getdoc(fn))
+    subargs, extra = parser.parse_known_args(args)
+    subcommands[subargs.subcommand](
+        subparsers.choices[subargs.subcommand],
+        extra)
+
+
+def _add(parser, args):
+    """ Add a blueprint to the Ambari server. """
+    parser.add_argument(
+        '--validate-topology',
+        default='true',
```

Comment:
Shouldn't that be a boolean:

```
parser.add_argument(
    '--validate-topology',
    default=True
    action='store_true',
    help="...")
```

Explicit is better than implicit--I like having the default in the code.